### ISSUE TITLE
SmartCity-Accessibility Server Side #567 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#339]
Build 'SmartCity-Accessibility Server Side' is failing!

Last 50 lines of build :
```
[...truncated 13.41 KB...]
 :: Spring Boot ::        (v1.5.2.RELEASE)

-- Product name: JxMaps
-- Licensed version: 1.x
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: May 15, 2017
JxMaps license valid.
Running smartcity.accessibility.services.NavigationServiceTest
Tests run: 3, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 27.125 sec <<< FAILURE! - in smartcity.accessibility.services.NavigationServiceTest
navigationUnauthorized(smartcity.accessibility.services.NavigationServiceTest)  Time elapsed: 1.341 sec  <<< FAILURE!
java.lang.AssertionError: Status expected:<401> but was:<400>
	at smartcity.accessibility.services.NavigationServiceTest.navigationUnauthorized(NavigationServiceTest.java:50)

Running smartcity.accessibility.navigation.NavigationTest
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 5.726 sec - in smartcity.accessibility.navigation.NavigationTest
Running smartcity.accessibility.database.ParseDatabaseTest
Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.548 sec - in smartcity.accessibility.database.ParseDatabaseTest

Results :

Failed tests: 
  NavigationServiceTest.navigationUnauthorized:50 Status expected:<401> but was:<400>

Tests run: 13, Failures: 1, Errors: 0, Skipped: 1

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:44 min
[INFO] Finished at: 2017-05-15T10:17:45+00:00
[INFO] Final Memory: 33M/205M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/workspace/SmartCity-Accessibility Server Side/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Alexander Kaplan] 26ed3a67e157881d5f9db9157bf519937bd63efa - remove some code smells and return test to original form

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/567/)

**Comment by kapalex** [comment]
@yaelAmitay , returned it to original form, `NavigationServiceTest.navigationUnauthorized` test is failing

**Comment by yaelAmitay** [comment]
I'll look at it

**Comment by yaelAmitay** [comment]
I fixed it anf it works on myy computer
but I see it's still running(more then 10 min), so maybe there is something wrong with jenkins?

**Comment by kapalex** [comment]
no, it's still running because you created a long test that runs for 10 minutes
combined with other tests its about 20 mins run

**Comment by yaelAmitay** [comment]
I know but it's been more then 10 min....
(however I'll make this test shorter)

okey now that you edit you answere.... I'll wait the 20 min to see

**Comment by kapalex** [comment]
You can look here at the progress
http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/569/console
it's currently on the long test for about 7 minutes, so a few more left

**Comment by kapalex** [comment]
It passed the test that failed before, so we're all good on the build most likely

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
